cmd/key_rotate.go

```diff
@@ -81,13 +81,18 @@ func BuildCmdRotateKey(w io.Writer, handler RotateKeyHandler, rf *RootFlags) *co
 		"",
 		"Path to the file containing the wallet's passphrase",
 	)
-	cmd.Flags().StringVarP(&f.PublicKey,
-		"pubkey", "k",
+	cmd.Flags().StringVarP(&f.NewPublicKey,
+		"new-pubkey", "",
```

Comment:
```suggestion
	cmd.Flags().StringVar(&f.NewPublicKey,
		"new-pubkey",
```

Reply:
We should use the right methods for flags without short version. Can you propagate this change to the commands below ?